<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudent</class>
 <widget class="QMainWindow" name="AddStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить ученика (РЭ)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>350</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="letter">
       <property name="placeholderText">
        <string>Литра класса</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="subject">
       <property name="placeholderText">
        <string>Предмет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="month">
       <property name="placeholderText">
        <string>Месяц</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name">
       <property name="placeholderText">
        <string>ФИ ученика</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="points">
       <property name="placeholderText">
        <string>Количество баллов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mark">
       <property name="placeholderText">
        <string>Оценка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="percent">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Процент выполнения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_b">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="back_b1">
       <property name="text">
        <string>Вернуться назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
